README.md

```diff
@@ -230,11 +230,11 @@ Running Tests
 2. cd RIAK_TEST and run commands
 	* make
 5. Download and put `.riak_test.config` in home directory from following link
-	https://gist.github.com/cmeiklejohn/d96d7167e1a8ed4b128d
+	https://gist.github.com/lythq/f979e11749a868fc33dd
```

Comment:
Yours still has the project name floppystore, but riak_test on Antidote is named antidote.
So you get an error if you haven't run riak_test on the previous code, and the new code isn't the one being tested.
I also added the test_paths setting, so you're less prone to skip that step.